lib/snapengage/test.js

```diff
@@ -77,5 +77,72 @@ describe('SnapEngage', function(){
         analytics.called(window.SnapABug.setUserEmail, 'name@example.com');
       });
     });
+
+    /**
+     * SnapEngage does not have an API for triggering events :/
```

Comment:
Probably better to just `TODO`ize them since you spent time writing the tests, haha